src/escpos/printer/network.py

```diff
@@ -110,16 +110,18 @@ def open(self, raise_not_found: bool = True) -> None:
                 return
         logging.info("Network printer enabled")
 
-    def _raw(self, msg):
+    def _raw(self, msg: bytes) -> None:
         """Print any command sent in raw format.
 
         :param msg: arbitrary code to be printed
         :type msg: bytes
         """
+        assert self.device
```

Comment:
What is the motivation behind this assertion? If I remember correctly, the device property should open the connection automatically on access.

Reply:
I use `mypy --strict`to lint my code; mypy can't know that self.device has been initialized somehow in the lifecycle of the object, it's the only way to tell it `yes it's fine`

Reply:
OK, agreed, that is a bit much to expect from a static code checker :-)